### ISSUE TITLE
Trade route through ice.

### ISSUE THREAD

**Issue by adaneslavo** [LoneGazebo/Community-Patch-DLL#4340]
1. Mod version 4-30

On cutom map where ice can be placed everywhere not only on poles, game creates trade routce through ice, even if normal units cannot cross those tiles.
![20180505000008_1](https://user-images.githubusercontent.com/33016300/39656016-6283e444-4ffd-11e8-89fd-f851d2e845bf.jpg)

**Comment by Iamblichos** [comment]
Yep if plot is owned, seatrades can pass through and owned boats—by city owner— can pass through.

**Comment by adaneslavo** [comment]
I didn't know that. But it is strange don't you think? I know this only the game and it needs sometimes simplifying, but I think ice should be tile were nothing can step on, maybe except all submarines.

**Comment by Iamblichos** [comment]
This is a design intent we made a long time ago.
